### PR TITLE
Add Sample

### PR DESCRIPTION
**Pull request by tnorling**
## Purpose

Add initial sample to repo

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[x] Other... Please describe: New Sample
```

## How to Test
Follow instructions in README

## What to Check
Verify that following the instructions successfully starts the sample and the sample works as expected.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,73 @@
-# Project Name
+# React Single-page Application built with MSAL React and Microsoft identity platform
```
Comment (derisen) [inline README.md]: lowercase: "React single-page application..."

File: README.md [hunk 2]
```diff
@@ -1,57 +1,73 @@
...
 
-(short, 1-3 sentenced, description of the project)
+This sample demonstrates how to use [MSAL React](https://www.npmjs.com/package/@azure/msal-react) to login, logout, conditionally render components to authenticated users, and acquire an access token for a protected resource such as Microsoft Graph.
 
 ## Features
```
Comment (derisen) [inline README.md:5]: possibly redundant given the summary above?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (derisen) [inline package.json:1]: a security vulnerability issue with one of the packages:

>  === npm audit security report ===                        
> 
> # Run  npm install react-scripts@4.0.1  to resolve 1 vulnerability
> SEMVER WARNING: Recommended action is a potentially breaking change
> 
>   High            Prototype Pollution                                           
> 
>   Package         object-path                                                   
> 
>   Dependency of   react-scripts                                                 
> 
>   Path            react-scripts > resolve-url-loader > adjust-sourcemap-loader  
>                   > object-path                                                 
> 
>   More info       https://npmjs.com/advisories/1573                             
> 
> 
> 
> found 1 high severity vulnerability in 1618 scanned packages
>   1 vulnerability requires semver-major dependency updates.

File: src/ui.jsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+            </UnauthenticatedTemplate>
+        </>
+    );
+};
+
+export const PageLayout = (props) => {
+    return (
+        <>
+            <Navbar bg="primary" variant="dark">
+                <a className="navbar-brand" href="/">MS Identity Platform</a>
```
Comment (derisen) [inline src/ui.jsx]: "Microsoft identity platform"

File: src/authConfig.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+        storeAuthStateInCookie: false, // Set this to "true" if you are having issues on IE11 or Edge
+    },
+    system: {	
+        loggerOptions: {	
+            loggerCallback: (level, message, containsPii) => {	
+                if (containsPii) {		
+                    return;		
+                }		
+                switch (level) {		
+                    case msal.LogLevel.Error:
```
Comment (derisen) [inline src/authConfig.js]:
```
Failed to compile.

./src/authConfig.js
  Line 23:26:  'msal' is not defined  no-undef
  Line 26:26:  'msal' is not defined  no-undef
  Line 29:26:  'msal' is not defined  no-undef
  Line 32:26:  'msal' is not defined  no-undef

Search for the keywords to learn more about each error.
```

instead 
```javascript
import { LogLevel } from "@azure/msal-browser";
```
then
```javascript
case LogLevel.Error:		
                        console.error(message);		
                        return;	
```

**Review by derisen — CHANGES_REQUESTED** [review]
Looks awesome! one issue and few nitpicks

File: README.md [hunk 6]
```diff
@@ -1,57 +1,73 @@
...
+## Running the sample
 
-(Add steps to start up the demo)
+1. Configure authentication and authorization parameters:
+   1. Open `src/authConfig.js`
+   2. Replace the string `"Enter_the_Application_Id_Here"` with your app/client ID on AAD Portal.
+   3. Replace the string `"Enter_the_Cloud_Instance_Id_HereEnter_the_Tenant_Info_Here"` with `"https://login.microsoftonline.com/common/"` (*note*: This is for multi-tenant applications located on the global Azure cloud. For more information, see the [documentation](https://docs.microsoft.com/azure/active-directory/develop/quickstart-v2-javascript-auth-code)).
+   4. Replace the string `"Enter_the_Redirect_Uri_Here"` with the redirect uri you setup on AAD Portal.
+2. Configure the parameters for calling MS Graph API:
+   2. Replace the string `"Enter_the_Graph_Endpoint_Herev1.0/me"` with `"https://graph.microsoft.com/v1.0/me"`.
```
Comment (derisen) [inline README.md]: maybe we can add a note here?

`(*note*: This is for MS Graph instance located on the global Azure cloud. For more information, see the [documentation](https://docs.microsoft.com/en-us/graph/deployments))`

File: README.md (hunk 2 above)
Comment (tnorling) [reply README.md:5]: The other samples have a blurb like this, I think it's fine to give a quick summary, but I'll give you final say.

File: README.md (hunk 1 above)
Comment (tnorling) [reply README.md]: Done

File: package.json (hunk 3 above)
Comment (tnorling) [reply package.json:1]: Done

File: src/authConfig.js (hunk 5 above)
Comment (tnorling) [reply src/authConfig.js]: Done

File: README.md (hunk 6 above)
Comment (tnorling) [reply README.md]: Done

File: src/ui.jsx (hunk 4 above)
Comment (tnorling) [reply src/ui.jsx]: Done

**Review by derisen — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (derisen) [reply README.md:5]: just a nitpick, let's keep it then!

**Comment by tnorling** [comment]
@derisen I don't have permission to merge, feel free to do so whenever you're ready

**Pull request merged**